src/players/index.js

```diff
@@ -24,4 +25,5 @@ export default [
   UstreamLive,
   FilePlayer,
   Iframe,
-]
+  VAST,
```

Comment:
does this work? Iframe returns canPlay true...

Reply:
not sure how this type is ever played?

Reply:
because the regex in Iframe fails on vast URLs

Reply:
wait but our iframe player just does `return true` i thought

Reply:
^ this.. not sure how VAST is ever played

Reply:
non issue anymore